helpdesk_mgmt_sla/views/helpdesk_sla_views.xml

```diff
@@ -0,0 +1,112 @@
+<odoo>
+    <record id="view_helpdesk_sla_search" model="ir.ui.view">
+        <field name="name">helpdesk.sla.form.search</field>
+        <field name="model">helpdesk.sla</field>
+        <field name="arch" type="xml">
+            <search string="Helpdesk SLA Search">
+                <filter
+                    string="Archived"
+                    name="inactive"
+                    domain="[('active','=',False)]"
+                />
+                <separator />
+                <field name="name" filter_domain="[('name', 'ilike', self)]" />
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="helpdesk_ticket_sla_form">
+        <field name="name">helpdesk.sla.form</field>
+        <field name="model">helpdesk.sla</field>
+        <field name="arch" type="xml">
+            <form string="SLA">
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            string="Archive"
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-archive"
+                            invisible="not active"
+                        />
+                        <button
+                            string="Restore"
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-archive"
+                            invisible="active"
+                        />
+                        <field name="active" invisible="1" />
+                    </div>
```

Comment:
This is outdated. We should use the web_ribbon approach here

```
<widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
```

There are plenty of examples in odoo for inspiration :) 
Essentially, you add the ribbon (after the button box / or before the title or any other form element), and then the action to archive/unarchive is done already by odoo through the action menu